File: Examples/DotNET/SDK/VisualBasic/Document/WordAndParagraphCount.vb
Imports Aspose.Cloud

Namespace Document
    Class WordAndParagraphCount
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Get Word and Paragraph Count from Document
            Dim WordsStatDataResponse As WordsStatDataResponse = Common.WordsService.WordsDocumentStatistics.ReadDocumentStatistics(inputfile, Common.FOLDER)

            Console.WriteLine((Convert.ToString(vbLf) & inputfile) + " have page count " + WordsStatDataResponse.StatData.PageCount + ", paragraph count " + WordsStatDataResponse.StatData.ParagraphCount + " and words count " + WordsStatDataResponse.StatData.WordCount)
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Document/WordAndParagraphCount.cs
using System;
using Aspose.Cloud;

namespace CSharp.Document
{
    class WordAndParagraphCount
    {
        public static void Run()
        {
            string inputfile = "doc-sample.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Get Word and Paragraph Count from Document
            WordsStatDataResponse WordsStatDataResponse = Common.WordsService.WordsDocumentStatistics.ReadDocumentStatistics(inputfile, Common.FOLDER);

            Console.WriteLine("\n" + inputfile + " have page count " + WordsStatDataResponse.StatData.PageCount + ", paragraph count " + WordsStatDataResponse.StatData.ParagraphCount + " and words count " + WordsStatDataResponse.StatData.WordCount);
        }
    }
}
